src/styles.css

```diff
@@ -0,0 +1,484 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Slab:wght@300;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100rem;
+  height: 89.625rem;
+  padding: 9.313rem 3.75rem 7.688rem 6.25rem;
+  background-color: #f5f6fa;
+}
+
+.page {
+  width: 87.5rem;
+  height: 68rem;
+  margin: auto;
+  padding: 0 0 23rem;
+  border-radius: 4px;
+  box-shadow: 40px 40px 100px 0 rgba(0, 0, 0, 0.2);
+  background-color: #fafafa;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav-container {
+  display: flex;
+  align-items: center;
+  height: 5.938rem;
+  padding: 0 6.25rem;
+  background-color: #fff;
+  margin-bottom: 3.125rem;
+  border-bottom: 1px solid #e8e8e8;
+}
+
+.title-app {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+  font-family: 'Montserrat', sans-serif;
+}
+
+nav ul {
+  display: flex;
```

Comment:
- [ ] Kindly make sure to not repeat the same CSS rule, you can select the most used rules like `display: flex` and create a single class for this e.g: `.flex` and apply this class to the HTML elements that use it

Reply:
@Ismailco Thank you for your review, but I have some doubts about it. I understand that there are some styles that are repeated in the different components, but it is a good practice also, not to have styles in the html, it is one of the reasons why I do not include Bootstrap. There is another way to approach repeating styles, what you suggest I didn't see in the microverse stuff and was never asked for before